runtime-light/tl/tl-types.cpp

```diff
@@ -53,6 +54,8 @@ bool GetPemCertInfoResponse::fetch(TLBuffer &tlb) noexcept {
   }
 
   auto response = array<mixed>::create();
+  constexpr size_t RESPONSE_SIZE = 10; // not more than
+  response.reserve(RESPONSE_SIZE, false);
```

Comment:
Why not just use `size` from the line 51?

Reply:
Yeah, fixed